src/app/app.component.html

```diff
@@ -1,10 +1,17 @@
 <div class="flex min-h-screen flex-col">
+  <div class="flex w-full justify-center">
+    <a
```

Comment:
You should add a short comment explaining that this is an a11y feature, because the next dev will wonder what it's supposed to do.